### PR TITLE
[PT FE]: support aten::log, aten::log2

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *support aten::log and aten::log2 in pytorch frontend*
 - *...*

### Tickets:
 - *101052*
 - *101094

### CODE REVIEW

File: src/frontends/pytorch/src/op/log.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    auto two = context.mark_node(opset10::Constant::create(element::f32, Shape{}, {2}));
+    if (x.get_element_type().is_integral()) {
+        x = context.mark_node(std::make_shared<opset10::Convert>(x, element::f32));
+    } else {
+        two = context.mark_node(std::make_shared<opset10::ConvertLike>(two, x));
+    }
+    auto log2 = context.mark_node(std::make_shared<opset10::Log>(two));
+    auto log = context.mark_node(std::make_shared<opset10::Log>(x));
+    auto res = context.mark_node(std::make_shared<opset10::Divide>(log, log2));
+    return {res};
```
Comment (rkazants) [inline src/frontends/pytorch/src/op/log.cpp:34]: are we sure that the output type is `fp32`? please check

File: src/frontends/pytorch/src/op/log.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_log(NodeContext& context) {
+    auto x = context.get_input(0);
```
Comment (rkazants) [inline src/frontends/pytorch/src/op/log.cpp:15]: please add a comment that it has a deal with natural logarithm as I know

File: src/frontends/pytorch/src/op/log.cpp (hunk 1 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/log.cpp:34]: tests covered this scenario, so yes I checked it

File: src/frontends/pytorch/src/op/log.cpp (hunk 2 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/log.cpp:15]: added

File: src/frontends/pytorch/src/op/log.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
```
[the remark this answers is not in the document]
Comment (eaidova) [reply src/frontends/pytorch/src/op/log.cpp]: removed

**Review by rkazants — APPROVED** [review]

File: src/frontends/pytorch/src/op/log.cpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/log.cpp:6]: Same comment about operations

**Review by mvafin — APPROVED** [review]

**Pull request merged**